src/components/SignUpForm.tsx

```diff
@@ -0,0 +1,150 @@
+import React, { useState } from 'react';
+import Form from 'react-bootstrap/Form';
+import Button from 'react-bootstrap/Button';
+
+function SignUpFormHeader() {
+  return <h3>Luo uusi käyttäjä</h3>;
+}
+
+function SignUpForm() {
+  const [values, setValues] = useState({
+    firstName: '',
+    lastName: '',
+    userId: '',
+    password: '',
+    passwordConfirm: '',
+  });
+
+  const handleFirstNameInputChange = (
```

Comment:
Add `useCallback` to avoid unnecessary renders https://reactjs.org/docs/hooks-reference.html#usecallback

Reply:
Same for each of the functions below